wemake_python_styleguide/visitors/base.py

```diff
@@ -271,3 +277,50 @@ def run(self) -> None:
         for token in self.file_tokens:
             self.visit(token)
         self._post_visit()
+
+
+class BaseCSTVisitor(BaseVisitor, CSTVisitor, metaclass=abc.ABCMeta):
+    """
+    Allows to store violations while traversing node cst.
+
+    This class should be used as a base class for all ``cst`` based checkers.
+    Method ``visit()`` is defined in ``CSTNode`` class.
+
+    Attributes:
+        tree: ``libcst.Module`` tree to be checked.
+        METADATA_DEPENDENCIES: ``Collection[ProviderT]`` metadata interface
```

Comment:
As far as I understand, the class inherits both libcst and WPS visitors. I think here it is not clear, that METADATA_DEPENDENCIES is libcst functionality, which is used implicitly by `self.tree.visit`, and this better be noted.

Reply:
I suggest we discuss it. why not clear. `BaseCSTVisitor` inherits `BaseVisitor` and `BaseVisitor` doesn't know anything about `self.tree`. But mb should be noted what used MetadataWrapper. what do u think about next text. 
`Around a ``libcst.Module`` is used ``MetadataWrapper``, that stores associated metadata for that module.`

Reply:
Suppose somebody not familiar with libcst, tries to make a plugin. I suppose he may not recognize, that `METADATA_DEPENDENCIES` is related with `CSTVisitor` at all.